discovery-infra/tests/config/global_configs.py

```diff
@@ -37,7 +36,8 @@ def _get_iso_download_path(cluster_name: str):
     def __post_init__(self):
         super().__post_init__()
         if self.cluster_name is None or isinstance(self.cluster_name, str):
-            self.cluster_name = ClusterName()
+            self.cluster_name = ClusterName()  # todo rm cluster_name after removing config.cluster_name dependencies
```

Comment:
Add `self.infra_env_name = InfraEnvName()`

Reply:
This is cluster config. it doesn't have `infra_env_name`